app/src/main/java/com/example/game/fragments/CommunityFragment.java

```diff
@@ -69,31 +64,24 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         ParseQuery<Subscription> subscriptionParseQuery = ParseQuery.getQuery(Subscription.class);
         subscriptionParseQuery.whereEqualTo(Subscription.KEY_COMMUNITY, community);
         subscriptionParseQuery.whereEqualTo(Subscription.KEY_USER, ParseUser.getCurrentUser());
-        subscriptionParseQuery.getFirstInBackground(new GetCallback<Subscription>() {
-            @Override
-            public void done(Subscription object, ParseException e) {
-                object.setInteractionCount(object.getInteractionCount().intValue() + 1);
-                object.saveInBackground();
-            }
+        subscriptionParseQuery.getFirstInBackground((object, e) -> {
+            object.setInteractionCount(object.getInteractionCount().intValue() + 1);
+            object.saveInBackground();
         });
+
         getQueryEvents(community);
     }
 
     private void getQueryEvents(Community community) {
```

Comment:
now that you could return community as null in line 57, should mark as @Nullable and handle accordingly.